src/lambda_streams/Animals.java

```diff
@@ -1,62 +1,103 @@
 package lambda_streams;
 
 import java.util.*;
+import java.util.stream.Collectors;
 
 class Animals {
+
     //This class will be using streams.
     static List<String> animals = Arrays.asList("peacoCK","rabbit","chiwawa","OranguTAN","vipeR","cobra","paNDa","bUffalo","DeeR","maLLard");
 
     public static void main(String[] args) {
         System.out.println("original animals : " + animals);
-
-        //clean up the animals array by using the capsFirst() method. follow instructions in the method definition. 
+        //clean up the animals array by using the capsFirst() method. follow instructions in the method definition.
         List<String> cleaned = capsFirst(animals, false);
         System.out.println(cleaned);
-       
         //do not modify these addAnimal() method invocations
         addAnimal("rEIndeeR");
         addAnimal("Platypus");
         addAnimal("frOg");
         addAnimal("lEOpArD");
         //---------------------------------------
-
         capsFirst(animals,true);
         System.out.println(animals);
-
         List<String> lowered = lowerFirst(animals,false);
         System.out.println(lowered);
-
-        
-
     }
 
     static List<String> capsFirst(List<String> animaList, boolean mutate) {
```

Comment:
The expectation is the method signature would be modified to include an additional parameter for a lambda, and that lambda would replace the previous implementation.